.github/workflows/call-backport-label-updater.yaml

```diff
@@ -0,0 +1,24 @@
+---
+    name: Call Backport Label Updater
+    on:
+      pull_request_target:
+        types:
+          - closed
+        branches:
+          - v1.15
+          - v1.14
+          - v1.13
+          - v1.12
```

Comment:
The other question I have: Let's say that this approach makes sense, and we merge this into main. When we go to create the v1.16 branch based on main, it'll have this workflow in there by default. This will mean that the workflow exists on both main and the older branch. In that case would GitHub run the workflow twice? Once for `main` and once for the new `v1.16` branch? Or would it only run one? Consequently, do we need to delete this workflow when creating the v1.16 branch in future?

Reply:
Answer from Fabio below: No, GitHub searches the target branch for the workflow to run. So adding this workflow into main would not cause it to trigger on stable branches. There's no risk for running the workflow twice.